README.md

```diff
@@ -13,6 +13,68 @@ to set up a local development copy of Lorax, check out the
 [CONTRIBUTING.md](https://github.com/mozilla/lorax/blob/master/CONTRIBUTING.md)
 file.
 
+## Map Data
+
+Map data is handled in a seperate process. It is inputed into a spreadsheet, then exported out using a script.
+
+World Map Data Conversion Instructions
+
+1. Select and copy all the data (including the header row) from the [Data Source](https://docs.google.com/spreadsheets/d/1jN7RGMeA3-fofx5IGu1KWodRLRVl324kQj9r_7Kc6yg/edit#gid=0).
+2. Visit 'Mr Data Convertor' - http://shancarter.github.io/mr-data-converter/
+3. Paste into input CSV space
+4. Ensure out put is set to 'JSON Properties'
+5. Paste content of JSON Properties into development environment.
+
+## Data Structure
+
+In the data/ folder there are two subfolders: base/ and i18n/.
+
+### Base
+
+The base/ folder contains all data that is consistent across languages. None of the data in this folder will be changed as a result of localization. 
+
+##### main.json
+Contains all issues, grouped by topic. Data for styling, type of infographic, tags, etc.
+
+##### countries.topo.json
+Contains the SVG data for drawing the world map.
+
+### i18N
```

Comment:
Do we still need i18n seeing all the strings live in bedrock or, are there some additional strings that needs to be translated in this repo?